src/http-api.ts

```diff
@@ -189,7 +189,7 @@ export class HttpApi<S extends Schema> extends EventEmitter {
    * @protected
    */
   async getResponseBody(response: HttpResponse) {
-    if (response.statusCode === 204) {
+    if (response.statusCode === 204 || response.statusCode === 201) {
```

Comment:
Heya @colincasey, why would we want to return no content for a 201?
201 is not a no-content reponse: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201. According to those docs it should return the newly crated resource in the body of the message.

Reply:
good question.  at one point *i was getting a 201 response with an empty body* but i can't remember what was triggering that now.  it may have been at an earlier stage when i hadn't figured out most of the implementation.  removing it now doesn't seem to affect the test suite so i'll revert this.